### PR TITLE
Add `Optional.Publisher` for Rx `from` operator

### PR DESCRIPTION
**Pull request by yuzushioh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ It's based on the following blog post: [https://medium.com/gett-engineering/rxsw
 | flatMap               | flatMap                                  |                                                                                                          |
 | flatMapFirst          | ❌                                        |                                                                                                          |
 | flatMapLatest         | switchToLatest                           |                                                                                                          |
-| from                  | ❌                                        |                                                                                                          |
+| from                  | Optional.Publisher(_ output:)            |                                                                                                          |
```
Comment (freak4pc) [inline README.md:79]: This is specifically 
```suggestion
| from(optional:)       | Optional.Publisher(_ output:)            |                                                                                                          |
```

The one I referred to was the regular Obsevable.from(_:) that takes an array or sequence - those could actually be reproduced by `.publisher`

**Comment by yuzushioh** [comment]
@freak4pc Thank you for the suggestion. I fixed it.

**Comment by freak4pc** [comment]
@yuzushioh thanks, can you run the script again? the image doesn't seem updated

**Comment by yuzushioh** [comment]
@freak4pc Done!

**Pull request merged**